package/MDAnalysis/analysis/base.py

```diff
@@ -40,17 +40,60 @@
 logger = logging.getLogger(__name__)
 
 
+class Results(dict):
+    r"""Container object for storing results.
+
+    Results are extend dictionaries by enabling values to be accessed by key,
+    `results["value_key"]`, or by an attribute, `results.value_key`. 
```

Comment:
In reST, python code needs double backticks
```suggestion
    by dict key ``results["value_key"]`` or by object attribute, ``results.value_key``. 
```